src/main/scala/com/databricks/labs/mosaic/gdal/MosaicGDAL.scala

```diff
@@ -70,12 +71,19 @@ object MosaicGDAL extends Logging {
         s"sudo cp $mosaicGDALAbsolutePath/libgdalalljni.so.30 $path/libgdalalljni.so.30".!!
     }
 
+    //noinspection ScalaStyle
     private def copyInitScript(path: String): Unit = {
         val destPath = Paths.get(path)
-        val script = readResourceBytes("/scripts/install-gdal-databricks.sh")
         if (!Files.exists(mosaicGDALPath)) Files.createDirectories(mosaicGDALPath)
         if (!Files.exists(destPath)) Files.createDirectories(destPath)
-        Files.write(Paths.get(s"$mosaicGDALAbsolutePath/mosaic-gdal-init.sh"), script)
+
+        val w = new PrintWriter(new File(s"$mosaicGDALAbsolutePath/mosaic-gdal-init.sh"))
+        val scriptLines = readResourceLines("/scripts/install-gdal-databricks.sh")
+        scriptLines
+          .map { x => if (x.contains("proxy")) s"# $x" else x }
```

Comment:
Why commenting the proxy line here instead of commenting it directly in `install-gdal-databricks.sh`?

Reply:
copy paste ooopsy, I will fix that